test-utils/test-utils-core/src/test/java/uk/gov/justice/services/test/utils/core/random/IntegerGeneratorTest.java

```diff
@@ -0,0 +1,218 @@
+package uk.gov.justice.services.test.utils.core.random;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class IntegerGeneratorTest {
```

Comment:
Need a test case which tests using Integer.MIN_VALUE and Integer.MAX_VALUE. This is to check that code handles the overflow case.